components/zcash_address/Cargo.toml

```diff
@@ -15,7 +15,7 @@ categories = ["cryptography::cryptocurrencies", "encoding"]
 keywords = ["zcash", "address", "sapling", "unified"]
 
 [dependencies]
-bech32 = "0.8"
+bech32 = "0.9"
```

Comment:
AFAICT this dependency is not exposed, so we can cut a point release of `zcash_address` at the same time we cut `zcash_primitives 0.11` et al.